app/views/site/index.html.erb

```diff
@@ -1,18 +1,17 @@
 <h1>
-  Welcome to Touchpoints
+  <%= t('site.header',touchpoints: (t 'touchpoints')) %>
 </h1>
 <p>
-  A <strong>Touchpoint</strong>
-  is the interaction point between a Customer and Service Provider.
+  <%= t('site.definition_html',touchpoint: (t 'touchpoint')) %>
 </p>
 <p>
-  Touchpoints exists to help Federal Agencies:
+  <%= t('site.list_header',touchpoints:(t 'touchpoints')) %>
 </p>
 <ul class="usa-list">
-  <li>Administer the A-11 Form</li>
-  <li>Streamline CX CAP Goal reporting</li>
-  <li>Solicit open-ended Feedback</li>
-  <li>Recruit users to talk to during product and service development</li>
+  <li><%= t 'site.list_item_1' %></li>
```

Comment:
`list_item` is a bit generic here. okay for now.

In the future, I could see a `product_features_list_item` or something a bit more descriptive, as `list_item` is generic enough to be used for any `li`